src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[],
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
+    <div className="movies" data-cy="card">
+      {movies.map(movie => (
+        <MovieCard
+          movie={movie}
+          key={movie.imdbId}
+        />
+      ))}
+    </div>
+  </>
```

Comment:
remove useless fragment
```suggestion
    <div className="movies" data-cy="card">
      {movies.map(movie => (
        <MovieCard
          movie={movie}
          key={movie.imdbId}
        />
      ))}
    </div>
```